bin/apps.bash

```diff
@@ -24,8 +24,6 @@ apps_run_sc() {
     log_info "Applying applications in service cluster"
 
     (
-        : "${scripts_path:?Missing scripts path}"
-        : "${secrets[kube_config_sc]:?Missing service cluster kubeconfig}"
```

Comment:
Why are these and all other similar lines removed? Is it to pass some shellcheck tests?

Reply:
They were introduced because shellcheck weren't able to see them in the sourced files so shellcheck complained about the referenced variables not being assigned. By using the `-x/--external-sources` flag with shellcheck and adding the `# shellcheck source=x` directive, shellcheck is now able to see the values defined in sourced files.

Reply:
Ah, now I understand